sites/public/src/pages/_app.tsx

```diff
@@ -42,8 +45,8 @@ function BloomApp({ Component, router, pageProps }: AppProps) {
   }, [])
 
   useMemo(() => {
-    // Need to add translations to both doorway uic and uic in order for
-    // both sets of components to properly translate.
+    // HACK ALERT: we need to add translations to both doorway uic and uic in
+    // order for both sets of components to properly translate.
```

Comment:
@lisunshiny does this mean we have to add all of the translations throughout the site to UIC? Or just ones that are part of the top menus?  If the former, should we copy all that content over as part of this PR?

Reply:
no, this code imports the translations from shared/helpers into both UIC libraries (also this code is untouched -- just updating a comment for consistency)

Reply:
Great! Thanks!